pontoon/base/views.py

```diff
@@ -988,50 +994,119 @@ def transvision(request):
         return HttpResponse("error")
 
 
+def download_file(prefixes, dirnames, relative_path):
+    for prefix in prefixes:
+        for dirname in dirnames:
+            url = os.path.join(prefix.replace('{locale_code}', dirname), relative_path)
```

Comment:
True! Interestingly, browsers seem to not care about duplicate slashes. I wonder if requests does?

I do wish Python had a urljoin method built in. _sigh_